bin/qc.py

```diff
@@ -59,6 +59,13 @@ def get_covered_pos(pos_depth, min_depth):
     
     return counter
 
+def get_depth_coverage(pos_depth, ref_length):
```

Comment:
does pos_depth  contain every position in the reference or just the ones with at least one read. Please double check.

Reply:
I believe it does. It creates a list that is [id, position, coverage] and has a length of 29903 (which matches the len of the ref sequence) when testing it again here

Reply:
Perfect. Thanks for checking